pkgs/development/libraries/webkitgtk/default.nix

```diff
@@ -222,6 +230,17 @@ stdenv.mkDerivation rec {
     sed 43i'#include <CommonCrypto/CommonCryptor.h>' -i Source/WTF/wtf/RandomDevice.cpp
   '';
 
+  postInstall = ''
+    mv $out/libexec/MiniBrowser $out/bin/MiniBrowser
```

Comment:
Why move it?

Reply:
To encourage people to actually run it when creating PRs.

It looks like I got the source path wrong however, and it's actually `libexec/webkit2gtk-4.0/MiniBrowser`. It took a while to figure out because of the time it takes to compile WebKit.

Reply:
I would worry it might also encourage people to use it, which might not be secure. Would not adding a comment asking people to do that achieve the same?

Reply:
Good point. I did a bit of research and it seems that sandboxing isn't enabled for MiniBrowser, so that would be problematic if people started using it for normal browsing. I'll put it back to `libexec` in the next commit.

I think we can reconsider this after Nixpkgs WebKitGTK has switched to GTK 4, though. With GTK 4, [sandboxing will become mandatory on WebKitGTK][1]. With that fixed, MiniBrowser should be secure as any other WebKitGTK-based browser. MiniBrowser itself just deals with GTK windows and dialogs, delegating rest of the work to `WebKitWebView`. All the security heavy-lifting should be done in WebKit, so as long as `WebKitWebView` is secure by default, I believe it's safe for normal use.

[1]: https://blogs.gnome.org/mcatanzaro/2020/03/31/sandboxing-webkitgtk-apps/